test/test_types_plugin.py

```diff
@@ -76,7 +76,7 @@ def test_float_modify_and_query(self):
 
         # Match in range
         out = self.list(u'myfloat:-10..0')
-        self.assertIn(b'aaa', out)
+        self.assertIn('aaa', out)
```

Comment:
I suppose these changes mean we need to decide whether the captured `stdout` stream in the tests should be Unicode, bytes, or "native" (i.e., Unicode on 3, bytes on 2). I don't have a strong feeling about what we choose, but each has different implications for how these checks go.